app/controllers/api/v1/messages_controller.rb

```diff
@@ -37,6 +37,8 @@ class MessagesController < Api::V1::ApiController
       param :recipient_id, String, desc: "Specific user id to fetch the messages of"
       param :state, String, desc: "Message state (unread|read) to filter on"
       def index
+        return invalid_messageable_resource if params[:messageable_id].present? && params[:messageable_type].blank?
```

Comment:
```suggestion
        raise Goodcity::InvalidParamsError.new('the.translation.key') if params[:messageable_id].present? && params[:messageable_type].blank?
```
To have a consistent error format